Dynatrace/dynatrace-operator
# Pushes release image to all registries

# Description

Extends the GitHub Actions to include pushing the image of a release to RHCC, GCR and DockerHub as well as creating multiarch manifests for them.

## How can this be tested?

* Create a release tag in the form v[0-9]+.[0-9]+.[0-9]+
* Check if the actions complete successfully

## Checklist
~~- [ ] Unit tests have been updated/added~~
